### PR TITLE
Flypie-305 update funnel cake blacklight app to bootstrap 5

### PR DESCRIPTION
**Pull request by cdoyle-temple**

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -70,7 +70,7 @@
   # config.i18n.raise_on_missing_translations = true
 
   # Annotate rendered view with file names.
-  # config.action_view.annotate_rendered_view_with_filenames = true
+  config.action_view.annotate_rendered_view_with_filenames = true
```
Comment (ebtoner) [inline config/environments/development.rb:73]: @cdoyle-temple -- Can you explain what this is needed for?

Comment (nomadicoder) [reply config/environments/development.rb:73]: @ebtoner This should make debugging templates and view partials easier.  We can see what's being rendered directly in the html. @cdoyle-temple ?

**Review by nomadicoder — APPROVED** [review]

File: config/environments/development.rb (hunk 1 above)
Comment (ebtoner) [reply config/environments/development.rb:73]: Ok, makes sense. Thanks for explaining!

Comment (cdoyle-temple) [reply config/environments/development.rb:73]: This helps when developing locally to see in the rendered code source (browser -> inspect) which template file generated each section of the page. Extremely handy shortcut that saves a lot of time with, especially with unfamiliar code-bases. We have it on manifold, the press, and library search.

**Pull request merged**